config/health_care_application.yml

```diff
@@ -0,0 +1,30 @@
+endpoints:
```

Comment:
I guess we have to change up application.yml to at least production/dev/test then, and maybe production can be set by Ansible? That way we can keep dev/test stuff in a place that makes sense as well, like how `config/saml.yml` does